storage/spanstore/downsampling_writer_test.go

```diff
@@ -81,7 +80,6 @@ func TestDownSamplingWriter_hashBytes(t *testing.T) {
 	}
 	_, _ = span.TraceID.MarshalTo(h.buffer)
 	// Same traceID should always be hashed to same uint64 in DownSamplingWriter.
-	assert.Equal(t, h.hashBytes(), h.hashBytes())
```

Comment:
The function comment is sensible, but the original test makes no sense, it didn't do that. Maybe fix it?